### PR TITLE
fix(i18n): Add some space to show the Locale values in the CM

### PR DESCRIPTION
**Pull request by simotae14**
### What does it do?

It adds more space (width) to show the locale content in the CM list view

### Why is it needed?

To solve an issue we have with the Locale column in the CM list view, the content is truncated with a 2rem width

### How to test it?

 - Create a new strapi project
 - Create a content type with i18n
 - Add an item to the content type
 - Check the content type overview table

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/issues/19790 and https://github.com/strapi/strapi/issues/20021

### CODE REVIEW

**Comment by lucasboilly** [comment]
Looks good to me!

File: packages/plugins/i18n/admin/src/components/LocaleListCell.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ const LocaleListCell = ({
         })}
       >
         <Button type="button" onClick={handleTogglePopover} ref={buttonRef}>
-          <ActionWrapper alignItems="center" justifyContent="center" height="2rem" width="2rem">
+          <ActionWrapper alignItems="center" justifyContent="center" height="2rem" width="10rem">
```
Comment (joshuaellis) [inline packages/plugins/i18n/admin/src/components/LocaleListCell.tsx:63]: Do we need the width even declared here? below we have a `maxWidth` set on the typography with ellipse, so the cell will just fit the content like we want without a set value?

**Review by joshuaellis — CHANGES_REQUESTED** [review]

File: packages/plugins/i18n/admin/src/components/LocaleListCell.tsx (hunk 1 above)
Comment (simotae14) [reply packages/plugins/i18n/admin/src/components/LocaleListCell.tsx:63]: Yes, you are right, we can use the maxWidth instead of the wrapper width

**Review by joshuaellis — APPROVED** [review]
less CSS === easy FE life

**Comment by simotae14** [comment]
The PR seems locked, I suppose a Github problem, but the code seems to be already merged in the main branch https://github.com/strapi/strapi/blob/450055e5afb80a784a3707acbcdacaea73021de9/packages/plugins/i18n/admin/src/components/LocaleListCell.tsx#L63 so I will close the PR

**Pull request closed without merging**